conda-envs/mlos_core-3.10.yml

```diff
@@ -28,5 +28,6 @@ dependencies:
     - bump2version
     - licenseheaders
     - pytest-local-badge
+    - sqlalchemy
```

Comment:
conda has SQLAlchemy 1.4, and we need v 2.0, hence pip

Reply:
Honestly not convinced conda is any better than just raw containers, but I guess we go where the devs are at.